### DISCUSSION TITLE
Can we use Bootstrap production version (bootstrap.min.css) loaded from node_modules as a seperate public asset in Webpack?

### DISCUSSION THREAD

**Discussion by haidiid** [webpack/webpack#18156] [Q&A]
Can we use Bootstrap production version (bootstrap.min.css) loaded from node_modules as a seperate public asset in Webpack? I hope so, so It won't litter the intended classess which are created by developer in seperate imported css files into a html document.

**Comment by alexander-akait** [comment]
You can use copy-webpack-plugin and just copy this file from node_modules
